en_US/README.md

```diff
@@ -1,21 +1,22 @@
 # Introduction (5.0 doc is under construction)
 
-*EMQX* is an open source MQTT message broker based on the Erlang/OTP platform.
+*EMQX* is an Open-Source, Cloud-Native, Distributed MQTT Broker for IoT based on the Erlang/OTP platform.
```

Comment:
Is it "Cloud-Native" though? Not really against the term just wondering if it applies here.

Sure, it can be launched in the cloud and we even have a cloud service...but the core software itself runs perfectly on local machines and company bare-metal/VM's as opposed to being cloud specific which implies on other people's hardware.

Again not against this if it's what we want to imply, just asking.

Reply:
This is a slogn that references the www.emqx.io. Extensibility is an important attribute. It can be better implemented based on Cloud-Native.